src/core/io/platform/f4xx/CANf4xx.cpp

```diff
@@ -0,0 +1,346 @@
+
+
+#include <cstring>
+#include <stdint.h>
+
+#include <HALf4/stm32f4xx.h>
+
+#include <core/io/pin.hpp>
+#include <core/io/platform/f4xx/CANf4xx.hpp>
+#include <core/io/platform/f4xx/GPIOf4xx.hpp>
+#include <core/platform/f4xx/stm32f4xx.hpp>
+#include <core/utils/log.hpp>
+#include <core/utils/time.hpp>
+#include <core/utils/types/FixedQueue.hpp>
+
+namespace {
```

Comment:
Is there a reason not to put this in the same namespace as the other stuff? You can wrap the anonymous namespace with `core::io` and it would behave the same

Reply:
Just following the organization standard of the other files in EVT-core. Might be better to put it in the `core::io` to avoid global name conflics.